client/src/screens/PersonalProfile/index.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component, Fragment } from 'react';
+import NavBar from '../../components/NavBar';
+import { Avatar, Grid, Typography } from '@material-ui/core';
+import { withStyles } from '@material-ui/styles';
+import style from './style';
+import Tabs from '../../components/Tabs';
+
+class PersonalProfile extends Component {
+	state = {
+		value: 0,
+	};
+
+	handleChange = (event, newValue) => {
+		this.setState({ value: newValue });
+	};
+
+	handleChangeIndex = (index) => {
+		this.setState({ value: index });
+	};
+	componentDidMount() {}
+	render() {
+		const { classes } = this.props;
+		return (
+			<Fragment>
+				<NavBar matchPath={this.props.match} history={this.props.history} />
+				<Grid
+					container
+					direction="column"
+					justify="flex-start"
+					alignItems="center"
+					className={classes.container}
```

Comment:
Tangential thought: One thing that's interesting about `material-ui` is that it exposes these CSS properties as individual component props. Sometimes it's tough to know how best to expose styles when writing reusable components that wrap several styled elements. Anyway interesting API design consideration